yt/visualization/_handlers.py

```diff
@@ -321,7 +320,11 @@ def get_norm(self, data: np.ndarray, *args, **kw) -> Normalize:
             dvmax = 1 * getattr(data, "units", 1)
         kw.setdefault("vmax", dvmax)
 
-        if self.norm_type is not None:
+        if data.ndim == 3:
+            assert data.shape[-1] == 4
+            # this is an RGBA array, only linear normalization makes sense here
+            norm_type = Normalize
```

Comment:
If it's RGBA then I'm not sure I understand in what cases we'd *want* normalization.  If it's not already scaled 0..1 or 0..255, then we need to apply that manually, and I don't think we want colorbars for that.

Reply:
indeed, this is just the simplest patch to restore previous behaviour. Before merging #3849 we used to pass a `Normalize` object in _init_image, and MPL didn't complain, because it's what it uses by default.
A different approach would be to patch this in `_init_image`, though I have a strong preference for leaving this responsability entirely to the `NormHandler` class because it's its sole purpose. Does it make sense ? Maybe I can add a little more details in the comment here ?